dev/AnimatedIcon/AnimatedIcon.h

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+#include "common.h"
+
+#include "AnimatedIcon.g.h"
+#include "AnimatedIcon.properties.h"
+
+class AnimatedIcon :
+    public ReferenceTracker<AnimatedIcon, DeriveFromPathIconHelper_base, winrt::AnimatedIcon>,
```

Comment:
I didn't write it out, there are 4 choices, the only real difference is the empty element that gets created within the parent grid. BitmapIcon creates an Image, FontIcon and SymbolIcon create a Textbox, while PathIcon creates a Path.  I chose PathIcon because I think path is the least expensive of those options.